src/include/compat.h

```diff
@@ -15,6 +15,7 @@
 #include "acconfig.h"
 
 #if defined(__linux__)
+#include <sys/types.h>
```

Comment:
nit, i think we should move this out of `#if defined(__linux__)` as `off_t` is part of POSIX standard.

Reply:
@tchaikov 
Sure